docs/streams/upgrade-guide.html

```diff
@@ -44,6 +44,12 @@ <h1>Upgrade Guide &amp; API Changes</h1>
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+
+    <h3><a id="streams_api_changes_110" href="#streams_api_changes_110">Streams API changes in 1.1.0</a></h3>
+    <p>
```

Comment:
Can we do this similar to https://github.com/apache/kafka/pull/4340/files#diff-4fddeffcac84ac672d984659747e0e90R93 ff ?

Reply:
Could you explain what you mean exactly by that? I do not fully understand what you intend here. Thanks. :)

Reply:
Sorry. The link is not correct... This one is better: https://github.com/apache/kafka/blob/trunk/docs/streams/upgrade-guide.html#L74-L79

We might want to list the newly added methods, too.